src/style.css

```diff
@@ -1,3 +1,80 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
CSS: Remember to use fallback fonts - alternative font-family in case the main one doesn't work